src/ed_gateway/webapi/controllers/driver_controller.py

```
@@ -111,11 +105,24 @@ async def claim_delivery_job(
+            driver_id=driver_id,
+            delivery_job_id=delivery_job_id,
+        )
     )
+
+
+async def _get_driver_id(user_id: str, mediator: Mediator) -> UUID:
+
+    response = (await mediator.send(GetDriverByUserIdQuery(user_id=user_id))).to_dict()
+
+    if not response["is_success"] or "data" not in response or "id" not in response["data"]:
+        raise ApplicationException(
```

Ruff (0.8.2)

Undefined name `ApplicationException`

(F821)